### DISCUSSION TITLE
Windows 11 Pro 23H2 - Windows 10 Start Menu Not Working

### DISCUSSION THREAD

**Discussion by ToddTaylor** [valinet/ExplorerPatcher#2308] [General]
Is there a known issue with the start menu only showing the Windows 11 version in Windows 11 23H2 (OS Build: 22631.2428)?

![image](https://github.com/valinet/ExplorerPatcher/assets/25310942/4b8a9ced-2876-4714-9b6a-cabc78461ad6)
![image](https://github.com/valinet/ExplorerPatcher/assets/25310942/da099a6a-0bd4-4442-83c4-2e8d3a669f82)
![image](https://github.com/valinet/ExplorerPatcher/assets/25310942/8fe98826-681f-443e-ba70-1b65871c137c)

**Comment by pyrates999** [comment]
Microsoft is starting to remove windows 10 features from windows 11 23h2.  One of those is the windows 10 start menu.

**Reply by ToddTaylor** [reply]
Thank you, @pyrates999.  I guess that means it's time to uninstall ExplorerPatcher and just start drinking the Microsoft Kook-Aid, no matter how much I hate the Windows 11 start menu.

**Reply by pyrates999** [reply]
You can try installing open shell to get the windows 7 start menu back: https://github.com/Open-Shell/Open-Shell-Menu

**Comment by Amrsatrio** [comment]
No. The old menu still exists in here. I'd suggest reinstalling EP and see because I tested it on my 22631.2428 laptop and it worked.

**Comment by Kimichouder** [comment]
if the start menu isnt working at ALL 
first go to task manager find explorer.exe(which is the background and taskbar)end task and run new task type in Explorer.exe and the start menu works again!

**Comment by FLKone** [comment]
Latest [pre-release](https://github.com/valinet/ExplorerPatcher/releases/tag/22621.2428.59.1_ac268b1) fixed the issue for me. Just reinstalled via **ep_setup.exe**.
